onnxmltools/convert/sklearn/shape_calculators/PCA.py

```diff
@@ -0,0 +1,31 @@
+# -------------------------------------------------------------------------
```

Comment:
This file is not necessary. Just copy `register_shape_calculator('SklearnPCA', calculate_sklearn_pca_output_shapes)` to this [place](https://github.com/onnx/onnxmltools/blob/0cd78a742837aa429c57d3a54f269a2421770cca/onnxmltools/convert/sklearn/shape_calculators/SVD.py#L31) because PCA and SVD are conceptually the same.